docs/sources/enterprise/saml/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: 'SAML authentication'
+description: 'Grafana SAML authentication'
+keywords: ['grafana', 'saml', 'documentation', 'saml-auth']
+aliases: ['/docs/grafana/latest/auth/saml/']
+weight: 10
+---
+
+# SAML authentication
+
+SAML authentication integration allows your Grafana users to log in by using an external SAML 2.0 Identity Provider (IdP). To enable this, Grafana becomes a Service Provider (SP) in the authentication flow, interacting with the IdP to exchange user information.
```

Comment:
General comment - the word 'enable' is preferred over the word 'allows'. Just a nit, but wanted to pass that along.